packages/landmarks-ppx/landmarks-ppx.1.4/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Preprocessor instrumenting code using the landmarks library"
+description: """
+Automatically or semi-automatically instrument your code using
+landmarks library."""
+maintainer: ["Marc Lasson <marc.lasson@lexifi.com>"]
+authors: ["Marc Lasson <marc.lasson@lexifi.com>"]
+license: "MIT"
+homepage: "https://github.com/LexiFi/landmarks"
+bug-reports: "https://github.com/LexiFi/landmarks/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "ppxlib" {>= "0.22"}
+  "landmarks" {= "1.4"}
```

Comment:
```suggestion
  "landmarks" {= version}
```
For next time, this is equivalent, and probably eases the maintainance burden for you

Reply:
Oh nice ! Thanks !